deck-o-cards.js

```diff
@@ -9,21 +9,35 @@ function deck_o_cards() {
 
   // Make 52 card objects and store them in the "cards" array
   // Hint: use 2 for loops
+  var cardNumb = 1;
 
 
-  // 2. Shuffle the cards
-  // Hint: shuffle function is already defined below
+  for (i=0; i < suits.length; i++){
+      var val = suits[i]
+     for(c=0; c < values.length; c++){
+         var numb = cardNumb++;
+         cards.push("card: " +numb+ ", value: " + values[c] + ", suit: " + val);
```

Comment:
You have to make sure to create the object with those properties, right now you're pushing strings into an array with not containing element to separate them (i.e. object {}).

Reply:
aha! exactly what I needed to see. thank you!